.github/workflows/ci.yml

```diff
@@ -0,0 +1,75 @@
+name: CI
+
+concurrency:
+  group: ci-${{ github.run_id }}
+  cancel-in-progress: true
+
+on:
+  push:
+    branches:
+      - main
+      - v[0-9]+
+      - v[0-9]+.[0-9]+
+      - cryostat-v[0-9]+.[0-9]+
+
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - labeled
+      - unlabeled
+    branches:
+      - main
+      - v[0-9]+
+      - v[0-9]+.[0-9]+
+      - cryostat-v[0-9]+.[0-9]+
+
+env:
+  TARGET_BRANCH: ${{ github.base_ref || github.ref }}
```

Comment:
I think we might be able to use `${{ github.base_ref || github.ref_name }}`.

Reply:
A little funny that `github.event.pull_request.base.ref` returns the short name. This is somehow nice as I got issues when running `ct lint` with full ref name :))